doit/settings/base.py

```diff
@@ -101,8 +101,7 @@
         'rest_framework.versioning.NamespaceVersioning'
```

Comment:
Burası hata vermiyor ama yinede sonuna virgül koyalım. Yada http://www.django-rest-framework.org/api-guide/versioning/#configuring-the-versioning-scheme burdaki gibi setleyelim.